src/unit_test.cu

```diff
@@ -1260,7 +1260,7 @@ void evolve_test(){
     par.store("write_file", false);
     par.store("write_it", true);
     par.store("energy_calc", true);
-    par.store("energy_calc_steps", 1000);
+    par.store("energy_calc_steps", 2000);
```

Comment:
For the final version of the unit test, should we increase the number of steps here?

Reply:
At 5000, it doesn't quite exit early since the difference between the 25000th and 20000th iterations is too large. I figured that 1000 was a bit too verbose, so 2000 would be satisfactory

Reply:
Fair enough. 2000 is fine, then